unixfs.go

```diff
@@ -187,6 +187,18 @@ func FSNodeFromBytes(b []byte) (*FSNode, error) {
 	return n, nil
 }
 
+// FSNodeFromBytesTFile unmarshal a protobuf message onto an FSNode.
+func FSNodeFromBytesTFile(b []byte) (*FSNode, error) {
+	n := new(FSNode)
+	err := proto.Unmarshal(b, &n.format)
+	*n.format.Type = TFile
+	if err != nil {
+		return nil, err
+	}
+
+	return n, nil
+}
+
 // NewFSNode creates a new FSNode structure with the given `dataType`.
```

Comment:
@bjzhang Maybe we could wrap `FSNodeFromBytes()` to implement it. I think it's pretty much the same logic as`FSNodeFromBytes()`.

Reply:
Sure, it is a quick patch for fixing the type issue. I will update it in my next push. Thanks.